src/app/view/components/dot-relationship-tree/dot-relationship-tree.component.scss

```diff
@@ -0,0 +1,114 @@
+@use "variables" as *;
+
+:host {
+    ul {
+        list-style: none;
+        padding-left: $spacing-5;
+        li {
+            span {
+                margin-left: $spacing-1;
+            }
+            .dot-tree__content {
+                display: flex;
+                align-items: center;
+            }
+        }
+    }
+    ul.dot-tree {
+        list-style: none;
+        padding: 0;
+        margin: 0;
+        position: relative;
+        & > li::after {
+            background: $gray-light;
+            width: 1px;
+            height: 19px;
+            content: "•";
+            position: absolute;
+            top: 19px;
+            left: 11px;
+            color: $gray-light;
+            display: flex;
+            justify-content: space-evenly;
+            align-items: end;
+            font-size: $font-size-x-large;
+            line-height: 1px;
+        }
+    }
+    ul.dot-tree--child {
+        position: relative;
+        & > li::after {
+            background: $gray-light;
+            width: 18px;
+            height: 1px;
+            content: "";
+            position: absolute;
+            left: 13px;
+            top: 11px;
+        }
+    }
+    ::ng-deep {
+        .dot-tree--active {
+            dot-icon {
+                position: relative;
+                &::after {
+                    background: $brand-primary;
+                    width: 7px;
+                    height: 7px;
+                    position: absolute;
+                    content: "";
+                    border-radius: 50%;
+                    right: 0;
+                    bottom: 0;
+                }
+            }
+        }
+        dot-icon-button {
+            padding: 0;
+            margin: 0;
+            background: none;
+            button.dot-icon-button {
```

Comment:
```suggestion
            button {
```

Reply:
@fmontes I need the `.dot-icon-button`

Reply:
Not sure I'm following here... we need it or not?